pkg/server/pod_controller.go

```diff
@@ -24,12 +24,18 @@ import (
 
 	"github.com/elotl/itzo/pkg/api"
 	"github.com/elotl/itzo/pkg/util"
-	"github.com/elotl/itzo/pkg/util/sets"
 	"github.com/golang/glog"
 	"golang.org/x/net/context"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	UPDATE_TYPE_NO_CHANGES = "no_changes"
+	UPDATE_TYPE_POD_RESTART = "pod_restart"
+	UPDATE_TYPE_POD_CREATE = "pod_created"
+	UPDATE_TYPE_UNITS_CHANGE = "units_changed"
```

Comment:
These seem like enums:
- The preferred way for enums in Go is via using `iota`.
- There's some mixed style for consts in itzo, but after a while we settled on using CamelCased variable names, not C/Python style UPPERCASE_WITH_UNDERSCORES.

Reply:
Yeah, I was thinking about using iota, but my original idea was to log change type as well, so string values were more suitable. I just forgot to add logging ;) added now + converted to CamelCase. Thanks!